src/content/blog/2025_werqshop_recap.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Operating at the edge of failure: WERQSHOP 2025 Recap"
+author: Nate Stemen
+day: 1
+month: 8
+year: 2025
+tags: 
+  - events
+  - error mitigation
+  - community
+  - mitiq
+---
+
+**WERQSHOP: Building a Community for Error Resilience in Quantum Computing**
+
+<div style="width:100%;">
+  <div style="float:right;width:40%;margin-left:2em;">
+    <figure>
+      <img class="not-prose" src="/images/2025_werqshop/sticker-nyu.png"/>
+      <figcaption><a href="https://werq.shop">WERQSHOP</a> was made possible by Unitary Foundation, NYU, and grants from the NSF and DOE.</figcaption>
+    </figure> 
+  </div>
+</div>
+
+On July 17<sup>th</sup> and 18<sup>th</sup> we convened 60 people from across the quantum computing ecosystem to discuss the current state and future of error mitigation.
+We brought together everyone from hardcore error correction theorists, to seasoned software maintainers working on mitigation tools.
+The goal was to bring a relatively small group of people together who have a vested interest in the topic to teach, learn, and discuss what's next.
+
+Depending on how you count it, error mitigation really came on the scene in 2017 with [_Error mitigation for short-depth quantum circuits_](https://arxiv.org/abs/1612.02058) and [_Efficient variational quantum simulator incorporating active error minimisation_](https://arxiv.org/abs/1611.09301) when early versions of <abbr title="Zero-Noise Extrapolation">ZNE</abbr> and <abbr title="Probabilistic Error Cancellation">PEC</abbr> were introduced.[^1]
+At that time, there was much hope and excitement surrounding <abbr title="Noisy Intermediate Scale Quantum">NISQ</abbr> devices, and <abbr title="Quantum Error Mitigation">QEM</abbr> techniques were seen as ways to boost their performance in the interim before we get fully fault-tolerant devices.
+We're now nearly a decade past the coining of the term error mitigation, and the environment in which these techniques were studied has shifted dramatically. 
+It's time to ask a new question:
+
+> What role does error mitigation play on early fault-tolerant computers?
+
+So that was exactly what we gathered people to discuss!
+
+[^1]: There are multiple other techniques that were invented and studied prior to ZNE and PEC in 2017 (for example, dynamical decoupling and decoherence-free subspaces), but these are generally not regarded as QEM techniques following Cai et al [arXiv:2210.00921](https://arxiv.org/abs/2210.00921).
```

Comment:
the arXiv link renders weirdly in the preview

Reply:
Had to escape the colon...[^1]

[^1]: Does it [work:here](https://werq.shop/)? Looks fine without the escaping in github markdown, so astro must be doing some funny parsing.